src/OctoshiftCLI.IntegrationTests/TestHelper.cs

```diff
@@ -113,6 +156,60 @@ public async Task ResetGithubTestEnvironment(string githubOrg)
             }
         }
 
+        public void InitializeBbsRepo(string bbsProjectKey, string repoName)
+        {
+            var repoPath = Path.Combine(Path.GetTempPath(), repoName);
+
+            if (Directory.Exists(repoPath))
+            {
+                Directory.Delete(repoPath, true);
+            }
+
+            Directory.CreateDirectory(repoPath);
+
+            Repository.Init(repoPath);
+            using var repo = new Repository(repoPath);
+            File.WriteAllText(Path.Join(repo.Info.WorkingDirectory, "README.md"), "# Test Repo");
+            repo.Index.Add("README.md");
+            repo.Index.Write();
+            var author = new Signature("Octoshift", "octoshift@github.com", DateTime.Now);
+            repo.Commit("Here's a commit i made!", author, author);
```

Comment:
We spoke about how you're interested in also creating PRs and such with the BBS API sometime in the future.  With this code, a unique commit ID will be created every time it is executed, which will also mean that we will need to use unique payloads to create the PRs.

We could use a Git fixture (that could even be stored as a repo in GitHub), but we could also go the full nine yards and use Atlassian's own migration tools to store test data :smile: This strategy would use the same archives that we're consuming in Octoshift, and we can use their first-party tools to do the work.

This has the potential to save us a lot of time in creating our test data.  In addition, we wouldn't need to use Git libraries, and it would reduce network requests made to set up the data.  We could also use [our own fixtures](https://github.com/github/octoshift/tree/129b1f77bda609df9c5bbc9e51d1ffba9c80ad4c/spec/fixtures/bitbucket_server) as test data, too.

Here is Atlassian's doc for migrating from and to BBS servers with migration archives: https://confluence.atlassian.com/bitbucketserver/migrating-bitbucket-server-to-another-server-776640406.html

This is certainly up for conversation and should definitely not be addressed in this PR, but I think there would be value in creating an issue to track this possibility :+1: